wiki/Tournaments/CWC/2023/en.md

```diff
@@ -110,39 +110,39 @@ The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/1c2
 
 | Team A | Team B | Match time | Stream link | |
```

Comment:
use "Twitch stream" in headers and remove "(Twitch)" from link texts

Reply:
addressed